students/Gabe_Lamberth/lesson02/music_gen.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+from pprint import pprint
+from pathlib import Path
+
+
+def main():
+    """
+        Using pathlib library to read in file assuming file and program store in same location
+        """
+    cwd = Path.cwd()
+
+    pth = cwd / 'featuresdf.csv'
+
+    # Load data
+    df = pd.read_csv("featuresdf.csv")
+
+    zip_df = zip(df.artists, df.name)
+
+    print("-----Part 1-----")
+    # Part 1 -- Generators
+    print(''.join([f'{artist}, {name}\n' for artist, name in zip_df if artist == 'Ed Sheeran']), end=' ')
+
+    # Part 2 -- closure
+    def music_function(df):
```

Comment:
use full words for your variable names.  Is it really that much work to type `data_frame` rather than `df` :)  I had to think about it to realize what it was.